core/services/gateway/handlers/workflow/handler.go

```diff
@@ -0,0 +1,91 @@
+package workflow
```

Comment:
@jinhoonbang is already adding this handler in his PR. Best to wait on that and then only extend with the Trigger method name (can be in a separate PR since it's technically part of CAPPL-21).

Reply:
I'll proceed with this in mine so that I'm not blocked if that's ok.